src/libs/actions/Workflow.ts

```diff
@@ -324,7 +326,9 @@ function clearApprovalWorkflow() {
     Onyx.set(ONYXKEYS.APPROVAL_WORKFLOW, null);
 }
 
-function validateApprovalWorkflow(approvalWorkflow: ApprovalWorkflowOnyx): Record<string, TranslationPaths> {
+type ApprovalWorkflowOnyxValidated = Omit<ApprovalWorkflowOnyx, 'approvers'> & {approvers: Approver[]};
```

Comment:
This is a trick that is later used whenever `validateApprovalWorkflow` is called in components, `approvalWorkflow` is narrowed down to `ApprovalWorkflowOnyxValidated` type